src/grt/src/GlobalRouter.cpp

```diff
@@ -628,10 +627,11 @@ void GlobalRouter::findPins(Net* net)
   }
 }
 
-void GlobalRouter::findPins(Net* net, std::vector<RoutePt>& pinsOnGrid)
+void GlobalRouter::findPins(Net* net, std::vector<RoutePt>& pinsOnGrid, int& root_idx)
 {
   findPins(net);
 
+  root_idx = -1;
```

Comment:
so if there is no driver you seg fault. better to use a valid default if you not going to check the result before using it